src/reader_util.cpp

```diff
@@ -13,6 +13,8 @@
 #ifdef LCF_SUPPORT_ICU
 #   include <unicode/ucsdet.h>
 #   include <unicode/ucnv.h>
+#	include <unicode/normalizer2.h>
```

Comment:
whitespace issues

Reply:
yep, I think I pasted in a tab by accident